packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -547,7 +547,7 @@ async function processCommonOpts({
 
   // https://webdriver.io/docs/options/#webdriver-options
   const webdriverOptions = {
-    logLevel: 'warn' as const, // info is super verbose right now
+    logLevel: 'trace' as const,
```

Comment:
It goes into a log file, so it turns out it isn't really the end of the world.

Reply:
Do we really need such detailed logging here?

Reply:
In reality they rarely use anything more than info, but also info is where all the good stuff happens. It becomes really useful when the only symptom is "webdriverio tried to start chromedriver and chromedriver tried to start the electron app and then nothing ever happened" and you want to see what it was doing or at least copy/paste the commands it was running to try and run it locally. Which is actually a common way things flake - I find myself doing that sort of thing multiple times a month.

It is out of the way in a file, anyway. Only applies to tests, doesn't seem to slow down the tests.. really handy do already have it when e2e tests flake. Then we can just download the electron-* files from the Files tab.

Reply:
Here's an example: https://mciuploads.s3.us-east-1.amazonaws.com/10gen-compass-main/1058ad5e3071bfdf69c62ac4e831a551da4c659a_leroux.bodenstein_919/smoketest-macos-x64-compass/smoketest-compasswdio.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAYDYF24CYHCWL5CHT%2F20241227%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20241227T165043Z&X-Amz-Expires=900&X-Amz-SignedHeaders=host&x-id=GetObject&X-Amz-Signature=22675107df4aed4d4ad15cecd5826b23531bbafc036dde75192c85610e95896e

See basically everything is INFO level.